mods/ts/rules/gdi-vehicles.yaml

```diff
@@ -145,10 +145,12 @@ HMEC:
 		Cost: 3000
 	Tooltip:
 		Name: Mammoth Mk. II
+		Description: Slow heavy walker.\n  Strong vs Infantry, Tanks and Aircraft\n  Weak vs Nothing\nMaximum one can be build.
```

Comment:
Ops, seems like git ate the `one` -> `1` change.
> Add "at a time" please

We do it like this everywhere else, so:
a) change it here and at several other places
b) let it as is
?